integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/OktaOrgCleaner.groovy

```diff
@@ -60,27 +63,81 @@ class OktaOrgCleaner {
 
         log.info("Deleting Groups:")
         client.listGroups().stream()
-                .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*")}
-                .forEach {
-                    log.info("\t ${it.getProfile().getName()}")
-                    it.delete()
-                }
+            .filter { it.getProfile().getName().matches(".*-${uuidRegex}.*") }
+            .forEach {
+                log.info("\t ${it.getProfile().getName()}")
+                it.delete()
+            }
 
         log.info("Deleting Group Rules:")
         client.listGroupRules().stream()
-                .filter { it.getName().matches("rule\\+${uuidRegex}.*")}
-                .forEach {
-                    GroupRule rule = it
-                    log.info("\t ${rule.getName()}")
-                    Util.ignoring(ResourceException) {
-                        rule.deactivate()
-                    }
-                    rule.delete()
+            .filter { it.getName().matches("rule\\+${uuidRegex}.*") }
+            .forEach {
+                GroupRule rule = it
+                log.info("\t ${rule.getName()}")
+                Util.ignoring(ResourceException) {
+                    rule.deactivate()
                 }
+                rule.delete()
+            }
 
         log.info("Deleting Policies:")
         client.listPolicies(PolicyType.OKTA_SIGN_ON.toString()).stream()
-            .filter { it.getName().matches("policy\\+${uuidRegex}.*")}
+            .filter { it.getName().matches("policy\\+${uuidRegex}.*") }
+            .forEach {
+                it.delete()
+            }
+
+        log.info("Deleting LinkedObjectDefinitions:")
+        client.listLinkedObjectDefinitions().stream()
+            .filter { it.getName().startsWith("manager") }
```

Comment:
same problem as before, if the test uses a pattern that could be a _real_ usage, the test should be updated (for example, if you need a _short_ name (for cases where the field is not long enough for a UUID, you could use `SDK-IT-*` or something, just update the IT in this PR

Reply:
sure, will update the PR once I refactor all the test object names.